api/commands/compile.ts

```diff
@@ -65,19 +65,17 @@ export async function compile(
 
 export class CompileChildProcess {
   public static forkProcess() {
-    // Runs the worker_bundle script we generate for the package (see packages/@dataform/cli/BUILD)
-    // if it exists, otherwise run the bazel compile loader target.
-    const findForkScript = () => {
-      try {
-        const workerBundlePath = require.resolve("./worker_bundle");
-        return workerBundlePath;
-      } catch (e) {
-        return require.resolve("../../sandbox/vm/compile_loader");
-      }
-    };
-    const forkScript = findForkScript();
+    let workerBundle: string;
+    try {
+      workerBundle = require.resolve("df/sandbox/vm/worker_bundle");
+    } catch (e) {
+      // The bundled CLI doesn't allow the `df` prefix; the source file is packaged directly.
+      workerBundle = "./sandbox/vm/worker_bundle";
```

Comment:
This won't work in the CLI, when you install `@dataform/cli` on a machine, this file won't exist.

This should be: `require.resolve("./worker_bundle")`

Additionally, you need to make sure the worker bundle js file get's copied to the CLI package. When the CLI runs on a normal machine it's executing inside a file such as:

`node_modules/@dataform/cli/bundle.js`

To make your require call work, you need to make sure there is also the file:

`node_modules/@dataform/cli/worker_bundle.js`

This may be as simple as keeping it as a dep of the CLI pkg_npm_tar rule, but you may also need to copy the file to the folder first. Let me know if that doesn't work on it's own.

Reply:
Copying worked, extracting shows this now!

```
$ tar -xvf bazel-out/k8-py2-fastbuild/bin/packages/@dataform/cli/package.tgz
package/
package/worker_bundle
package/bundle.js
package/package.json
```